manifests/quick-start-mysql.yaml

```diff
@@ -128,6 +135,8 @@ rules:
   resources:
   - workflowtemplates
   - workflowtemplates/finalizers
+  - clusterworkflowtemplates
```

Comment:
@sarabala1979 seem to be missing on master

Reply:
Merge from the latest master. This resource needs to be added as a ClusterRole and ClusterRoleBinding